rhizome/postgres/lib/hugepages_setup.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require_relative "../../common/lib/util"
+
+class HugepagesSetup
+  def initialize(instance)
+    @version, @cluster = instance.split("-", 2)
+  end
+
+  def hugepage_size_kib
+    r('awk \'/Hugepagesize/ {printf "%.2f\n", $2}\' /proc/meminfo').to_f
```

Comment:
Weird: awk? floating point?

```suggestion
    Integer(File.read('/proc/meminfo')[/^Hugepagesize:\s*(\d+)\s*kB/, 1], 10)
```

Reply:
You can test with:

```sh
$ ruby -e "puts Integer(File.read('/proc/meminfo')[/^Hugepagesize:\s*(\d+)\s*kB/, 1], 10)"
2048
```

On a system with 1GB hugpages:

```sh
$ ruby -e  ruby -e "puts Integer(File.read('/proc/meminfo')[/^Hugepagesize:\s*(\d+)\s*kB/, 1], 10)"
1048576
```